katdal/test/test_datasources.py

```diff
@@ -23,6 +23,7 @@
 
 import numpy as np
 from nose.tools import assert_raises
+
```

Comment:
Technically there should not be an empty line here. These packages are all in the category of "external dependencies", i.e. not in the standard library and not in `katdal`.